src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -17,29 +17,66 @@ Statuses:
+| [Select](/components/select) ~ new          | [available] |
+| [StartEnd](/components/start-end)           | [available] |
+| [TextArea](/components/text-area)           | [available] |
+| [TextInput](/components/text-input)         | [available] |
+| [ThemeProvider](/components/theme-provider) | [available] |
+| [Tooltip](/components/tooltip)              | [available] |
+| Grid                                        | [planned]   |
+| Toggle                                      | [planned]   |
+| Table                                       |             |
+| DatePicker                                  |             |
+| Navigation                                  |             |
+| Pagination                                  |             |
+| Tabs                                        |             |
+| Notification                                |             |
+| Text                                        |             |
+| Heading                                     |             |
+| Progress                                    |             |
+| TimePicker                                  |             |
+| Dialog                                      |             |
+| Tile Grid                                   |             |
+| Column Grid                                 |             |
+| List                                        |             |
+| ControlGroup                                |             |
+| StackedLayout                               |             |
+| Tag                                         |             |
+| Badge                                       |             |
+| TypeAhead                                   |             |
+| Toolbar                                     |             |
+| Detail                                      |             |
+| Alert                                       |             |
+| SidePanel                                   |             |
+| Wizard                                      |             |
+| ColorPicker                                 |             |
+| FlagLayout                                  |             |
+| Scrollable                                  |             |
+| BreadCrumbs                                 |             |
+| PageTemplate                                |             |
+| Sliders                                     |             |
+| Search                                      |             |
+| Truncation                                  |             |
+| Tree                                        |             |
+| Charts                                      |             |
+| Maps                                        |             |
+| NumberInput                                 |             |
+| Fix                                         |             |
+| DataTable                                   |             |
+| LogIn                                       |             |
```

Comment:
We talked about excluding Login & DataTable from our list entirely (and maybe even being declarative about not doing them), but not sure if we made a decision about it.